mq/main/bridge/bridge-api/src/main/java/com/sun/messaging/bridge/api/StompFrameMessage.java

```diff
@@ -423,11 +434,28 @@ public ByteBufferWrapper marshall(Object obj) throws IOException {
         }
     }
 
+
+    static String unescapeSpecialChars(String keyOrValue) {
```

Comment:
Please add unit tests for this method.

Reply:
I will add tests for this method and the escape counterpart